src/components/include/utils/logger2.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include <log4cxx/logger.h>
+#include <string>
+#include "utils/ilogger.h"
+
+namespace logger {
+class Log4CXXLogger;
+}
+
+typedef logger::Log4CXXLogger ExternalLogger;
+
+#define LOCATTION_INFO LOG4CXX_LOCATION
+
+#define CREATE_LOGGERPTR(logger_name)      \
+  namespace {                              \
+  static std::string logger_(logger_name); \
+  }
+
+#define LOG_WITH_LEVEL2(logLevel, logEvent)                               \
```

Comment:
Looks like this is some temporary file. Will skip it

Reply:
renamed